src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseSearchValuesTests.cs

```diff
@@ -0,0 +1,1023 @@
+
+                    private void TestMethod(ReadOnlySpan<char> chars)
+                    {
+                        _ = chars.IndexOfAny(s_myChars4);
+                    }
+                }
+                """;
+
+            await VerifyCodeFixAsync(LanguageVersion.CSharp7_3, source, expected);
+        }
+
+        [Fact]
+        public async Task TestCodeFixerWorksInTopLevelStatementsDocument()
+        {
+            string source =
+                """
+                using System.Buffers;
+
+                _ = "".IndexOfAny([|"aeiouA".ToCharArray()|]);
+                """;
+
+            string expected =
+                """
+                using System.Buffers;
+                using System;
+
+                _ = "".AsSpan().IndexOfAny(s_myChars);
+
+                partial class Program
+                {
+                    private static readonly SearchValues<char> s_myChars = SearchValues.Create("aeiouA");
+                }
+                """;
+
+            await VerifyCodeFixAsync(LanguageVersion.CSharp9, source, expected, topLevelStatements: true);
+
+            source =
+                """
+                using System.Buffers;
+
+                _ = "".IndexOfAny([|s_myValues|]);
+
+                partial class Program
+                {
+                    private static readonly char[] s_myValues  = new[] { 'a', 'e', 'i', 'o', 'u', 'A' };
+                }
+                """;
+
+            expected =
+                """
+                using System.Buffers;
+                using System;
+
+                _ = "".AsSpan().IndexOfAny(s_myValues);
+
+                partial class Program
+                {
+                }
+
+                partial class Program
+                {
+                    private static readonly SearchValues<char> s_myValues = SearchValues.Create("aeiouA");
+                }
+                """;
+
+            await VerifyCodeFixAsync(LanguageVersion.CSharp9, source, expected, topLevelStatements: true);
+        }
+
+        private static async Task VerifyAnalyzerAsync(LanguageVersion languageVersion, string source) =>
+            await VerifyCodeFixAsync(languageVersion, source, expected: null);
+
+        private static async Task VerifyCodeFixAsync(LanguageVersion languageVersion, string source, string expected, bool topLevelStatements = false)
+        {
+            await new VerifyCS.Test
+            {
+                ReferenceAssemblies = Net80,
+                LanguageVersion = languageVersion,
+                TestCode = source,
+                FixedCode = expected,
+                TestState = { OutputKind = topLevelStatements ? OutputKind.ConsoleApplication : null },
+            }.RunAsync();
+        }
+
+        // TEMP - need newer version of Microsoft.CodeAnalysis.Analyzer.Testing
+        // Replace with 'ReferenceAssemblies.Net.Net80'
+        private static readonly Lazy<ReferenceAssemblies> _lazyNet80 =
```

Comment:
Tagging @sharwell

Reply:
Maybe file an issue to replace it when `ReferenceAssemblies.Net.Net80` added?

Reply:
#6923